packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -197,12 +142,8 @@ const helpers = {
               ...contentNegotiationResult,
             }));
         } else {
-          logger.trace(`Unable to find a content for ${mediaTypes}, returning an empty text/plain response.`);
```

Comment:
Hey I posted a comment on this before but its gone, and I talked about it on Slack, this old behavior was correct. Let mew see if I can dig it out to avoid writing again.

Reply:
Ok, I guess it was lost because of the rebases!

Reply:
Here it is https://github.com/stoplightio/prism/pull/458#discussion_r303908700

I see a reply on there @XVincentX im not really sure what left and right are all about, but I'll be happy to learn.

Regardless of that, the code is still changing. Being more functional was not the issue. 

The issue is the removal of the old text/plan message. This behavior _is wanted_ for when X status is requested and X status is defined in responses but it just so happens to not have any content (no example(s) or schema).  Not having content **must not** cause a 406 error, this change has now become a bug.

Reply:
@philsturgeon The code is correct. This is *still* happening [here](https://github.com/stoplightio/prism/blob/master/packages/http/src/mocker/negotiator/NegotiatorHelpers.ts#L158-L167)

This part of the code happens when *there's a content*, but not the one you're requesting.

Does that clarify?

Reply:
`left`: something went wrong
`right`: I was able to do what you asked me to do

Reply:
Awesome, thank you!